### ISSUE TITLE
Quick acceptance times (< 5 days) for some respiratory journals

### ISSUE THREAD

**Issue by andreasronit** [dhimmel/delays#1]
Hey Daniel. In the data I find some journals with papers that have a very low acceptance time, such as one or two days. Do you have an explanation for that? Could it be Invited articles or other article types than original articles, such as editorials and correspondences? Andreas

**Comment by dhimmel** [comment]
Yeah, I've noticed this as well. There could be a few reasons behind such quick acceptance:
1. The articles are not subject to peer review, such as "editorials and correspondences"
2. The publisher reported incorrect dates
3. The journal "resets the clock" so the reported submission date corresponds to a revised manuscript being uploaded that doesn't need to go back to reviewers.

What journal(s) did you notice this for? One thing I like to do is find the actual PubMed IDs and then look at the articles to see which reason is most probable.

**Comment by andreasronit** [comment]
I think you are right. As you know I have been looking at these data for respiratory journals with IF >3, and it is only rather few (1%) that have acceptance time below 5 days of all these papers (N>18000). The journals include with some very low acceptance times include Thorax, Resp Res, and Resp Med among others.

**Comment by dhimmel** [comment]
> it is only rather few (1%) that have acceptance time below 5 days

Agreed, when I look at the plots for these journals, there are some super quick acceptances, but they fit with the overall distribution of delays. This makes me think that it's probably not a reporting error. For example, see:

![Respiratory Medicine Acceptance Delays](https://raw.githubusercontent.com/dhimmel/delays/59310065e325c71e7d121be240a0a3bc60875cc4/viz/journal/accept/8908438.png)

So here are the actual articles plotted above with acceptance in under 5 days:

| pubmed_id | delay_type | date | delay |
| --- | --- | --- | --- |
| 15878654 | Acceptance | 2005-03-06 | 3 |
| 15890507 | Acceptance | 2005-03-17 | 1 |
| 15894480 | Acceptance | 2005-04-11 | 4 |
| 15936184 | Acceptance | 2005-04-21 | 3 |
| 16039839 | Acceptance | 2005-06-10 | 1 |
| 16242311 | Acceptance | 2005-09-07 | 2 |
| 16263254 | Acceptance | 2005-09-23 | 2 |
| 16289785 | Acceptance | 2005-08-24 | 2 |
| 16338596 | Acceptance | 2005-03-23 | 1 |
| 16412623 | Acceptance | 2005-11-25 | 2 |
| 16626955 | Acceptance | 2006-02-28 | 1 |
| 16962307 | Acceptance | 2006-07-20 | 1 |
| 22917808 | Acceptance | 2012-08-02 | 1 |

`15878654` [is a SHORT COMMUNICATION](https://doi.org/10.1016/j.rmed.2005.03.016). `15890507` looks like an [article](https://doi.org/10.1016/j.rmed.2005.04.002) where the dates were reported correctly. There's definitely some interesting detective work to do!

**Comment by andreasronit** [comment]
Great. I stumbled upon [this report](http://publishingresearchconsortium.com/index.php/112-prc-projects/research-reports/peer-review-in-scholarly-journals-research-report/142-peer-review-in-scholarly-journals-perspective-of-the-scholarly-community-an-international-study) on attitudes to peer review in the scientific community that you may find interesting.
